DigitalLearningSolutions.Web/Views/ForgotPassword/ForgotPassword.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.ForgotPassword
+@model ForgotPasswordViewModel
+@{
+  ViewData["Title"] = "Digital Learning Solutions - Reset Your Password";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">Reset your password</h1>
+
+    <form method="post" role="form" asp-action="SubmitEmail">
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role seem to imply that role="form" is for div elements that resemble a form. Is there a reason to add it to the form tag?